<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PvRMWindow</class>
 <widget class="QDialog" name="PvRMWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_turn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>White's turn</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>480</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QPushButton" name="pushButton_goBack">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>goBack</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_actionCompleted">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>280</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Done!</string>
   </property>
  </widget>
  <widget class="QLabel" name="hidden_Label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton__giveUp">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>340</y>
     <width>293</width>
     <height>33</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Give up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sendUart">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>sendUart</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_e5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>e5</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
